### DISCUSSION TITLE
q&a

### DISCUSSION THREAD

**Discussion by guiworldtv2** [LITUATUI/M3UPT#15] [Q&A]
como consegues agendar a atualização dos links?
tens um linux e faz atraves deste ou consegues fazer remotamente com a computadora desligada?

**Comment by LITUATUI** [comment]
Antigamente usava o meu servidor (VPS) na Hetzner com um cron job, agora uso o GitHub Actions para isso.

Uso [workflows](https://github.com/LITUATUI/M3UPT/tree/main/.github/workflows) no GitHub tanto para actualizar o EPG como os streams dos canais TVI.

Aconselho [esta](https://github.com/features/actions) leitura.

**Reply by guiworldtv2** [reply]
ja esta a correr o arquivo base de demonstração .yml
os arquivos epg e stream precisam estar ligados a este para correr também?
espero não estar a incomodar, não estou a entender o que está acontecer

**Comment by LITUATUI** [comment]
Não percebi a pergunta.

São apenas workflows que executam scripts de x em x tempo.

**Reply by guiworldtv2** [reply]
estava apenas a perguntar como fazer correr os arquivos no git actions
até agora parece estar a funcionar bem

**Reply by guiworldtv2** [reply]
agradeço-lhe as ajudas que me dastes, estou a começar a aprender como funciona o mecanismos de listas m3u
